index.html

```diff
@@ -540,6 +549,7 @@
 <canvas class="Canvas" id="canvas"></canvas> <!--touch-action="none"-->
 <div class="Canvas" id="p5WebGL" style="position: absolute; left: 0; top: 0;"></div> <!--touch-action="none"-->
 <canvas id="glcanvas"></canvas>
+<canvas class="Canvas" id="testcan" style="position: absolute; left: 0; top: 0; transform: scale(0.5); display: none"></canvas> <!--touch-action="none"-->
```

Comment:
Do we still need this? Looks like what it was used for may have been removed

Reply:
Nope, removed. Thanks for pointing this out!